src/pages/home/report/ParticipantLocalTime.tsx

```diff
@@ -1,24 +1,23 @@
-import lodashGet from 'lodash/get';
 import React, {useEffect, useState} from 'react';
 import {View} from 'react-native';
-import participantPropTypes from '@components/participantPropTypes';
 import Text from '@components/Text';
-import withLocalize, {withLocalizePropTypes} from '@components/withLocalize';
+import useLocalize from '@hooks/useLocalize';
 import useThemeStyles from '@hooks/useThemeStyles';
 import DateUtils from '@libs/DateUtils';
 import Timers from '@libs/Timers';
+import type {Locale} from '@src/components/LocaleContextProvider';
 import CONST from '@src/CONST';
+import type {PersonalDetails} from '@src/types/onyx';
 
-const propTypes = {
+type ParticipantLocalTimeProps = {
     /** Personal details of the participant */
-    participant: participantPropTypes.isRequired,
-
-    ...withLocalizePropTypes,
+    participant: PersonalDetails;
 };
 
-function getParticipantLocalTime(participant, preferredLocale) {
-    const reportRecipientTimezone = lodashGet(participant, 'timezone', CONST.DEFAULT_TIME_ZONE);
-    const reportTimezone = DateUtils.getLocalDateFromDatetime(preferredLocale, null, reportRecipientTimezone.selected);
+function getParticipantLocalTime(participant: PersonalDetails, preferredLocale: Locale) {
+    // eslint-disable-next-line @typescript-eslint/prefer-nullish-coalescing -- Disabling this line for safeness as nullish coalescing works only if the value is undefined or null
+    const reportRecipientTimezone = participant.timezone || CONST.DEFAULT_TIME_ZONE;
```

Comment:
NAB 
```suggestion
    const reportRecipientTimezone = (participant.timezone ?? '') || CONST.DEFAULT_TIME_ZONE;
```

Reply:
@luacmartins I am not sure this is needed , if `participant.timezone` will be undefined or `''` the variable will be set to `CONST.DEFAULT_TIME_ZONE` so adding `?? ''` seems to me redundant

Reply:
yea, I agree. It was just a suggestion so we don't need to suppress the lint rule